src/GafferImageUI/ImageView.cpp

```diff
@@ -254,6 +276,9 @@ ImageView::ImageView( const std::string &name )
 	viewportGadget()->setPrimaryChild( m_imageGadget );
 
 	m_colorInspector = shared_ptr<ColorInspector>( new ColorInspector( this ) );
+
+	m_zoomLevelChangedConnection = zoomLevelChangedSignal().connect( boost::bind( &ImageView::setZoomLevel, this, ::_1) );
```

Comment:
I'm using a scoped connection here but I find rather annoying that there is no way to query a signal about its connection after the fact...